### ISSUE TITLE
Screen reader announces two role as 'tab' and 'list' item for History/Memory tab item.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1669]
Repro steps:
Open the Application Calculator
Navigate to ‘Standard’ button and activate it.
Navigate to the ‘History/Memory’ tab present at the left of the page.
Observe the issue.

Actual Result:
Screen reader announce two role tab/list item for History/Memory tab item.

Expected Result:
Screen reader shouldn't announce two role tab/list item for History/Memory tab item. Only one role as tab item should be defined for ‘History/Memory’ controls and Focus should move separately to 'History' and 'Memory' tab item for announcing correct role for both tab items.
